spec/lib/yaml_normalizer_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+require 'yaml_normalizer'
+
+RSpec.describe YamlNormalizer do
+  describe '#run' do
+    let(:tempfile) { Tempfile.new }
+
+    before do
+      File.open(tempfile.path, 'w') do |f|
+        f.puts <<~YAML
+          some:
+            key: >
+              quoted
+            value1: 'quoted'
+            value2: "quoted"
+        YAML
+      end
+    end
+
+    after { tempfile.unlink }
+
+    it 'normalizes a YAML files in-place' do
+      YamlNormalizer.run([tempfile.path])
+
+      expect(File.read(tempfile.path)).to eq <<~YAML
+        ---
+        some:
+          key: quoted
+          value1: quoted
+          value2: quoted
+      YAML
+    end
+  end
+
+  describe '#strip_each' do
```

Comment:
Yeah this is a result of me flip-flopping on instance vs class, I'll fix it, thanks.